x/shield/types/types.go

```diff
@@ -6,20 +6,43 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// Pool contains a shield pool's data.
 type Pool struct {
-	PoolID          uint64         `json:"pool_id" yaml:"pool_id"`
-	Active          bool           `json:"active" yaml:"active"`
-	Description     string         `json:"description" yaml:"description"`
-	Sponsor         string         `json:"sponsor" yaml:"sponsor"`
-	SponsorAddr     sdk.AccAddress `json:"sponsor_address" yaml:"sponsor_address"`
-	Premium         MixedDecCoins  `json:"premium" yaml:"premium"`
-	TotalCollateral sdk.Int        `json:"total_collateral" yaml:"total_collateral"`
-	Available       sdk.Int        `json:"available" yaml:"available"`
-	TotalLocked     sdk.Int        `json:"total_locked" yaml:"total_locked"`
-	Shield          sdk.Coins      `json:"shield" yaml:"shield"`
-	EndTime         time.Time      `json:"end_time" yaml:"end_time"`
+	// PoolID is the id of the pool.
+	PoolID uint64 `json:"pool_id" yaml:"pool_id"`
+
+	// Description is the term of the pool.
+	Description string `json:"description" yaml:"description"`
+
+	// Sponsor is the project owner of the pool.
+	Sponsor string `json:"sponsor" yaml:"sponsor"`
+
+	// SponsorAddr is the CertiK Chain address of the sponsor.
+	SponsorAddr sdk.AccAddress `json:"sponsor_address" yaml:"sponsor_address"`
+
+	// Active means new purchases are allowed.
+	Active bool `json:"active" yaml:"active"`
+
+	// Premium is the undistributed pool
```

Comment:
undistributed pool premium?

Reply:
What's the "correct" term Daryl suggested?

Reply:
It's service fee. Should we also update it in code?